libpromises/var_expressions.c

```diff
@@ -329,10 +339,32 @@ VarRef *VarRefParseFromNamespaceAndScope(const char *qualified_name,
     ref->lval = lval;
     ref->indices = indices;
     ref->num_indices = num_indices;
-
     return ref;
 }
 
+/*
+ * This function will return true if the given variable is
+ * a this.something variable that is an alias to a non-special local variable.
+ */
+bool is_this_not_special(const char *scope, const char *lval) {
+    // TODO: better way to get this list?
+    const char *special_this_variables[] = {"v","k","this","service_policy","promiser","promiser_uid","promiser_gid","promiser_pid","promiser_ppid","bundle","handle","namespace","promise_filename","promise_dirname","promise_linenumber"};
```

Comment:
Do you know why these variables names are not all considered as [reserved](https://github.com/cfengine/core/blob/master/libpromises/syntax.c#L815)?